src/universal/modules/teamDashboard/components/TaskColumn/TaskColumn.js

```diff
@@ -258,8 +257,12 @@ const styleThunk = () => ({
     color: appTheme.palette.dark,
     display: 'flex !important',
     lineHeight: '1.5rem',
-    padding: '.625rem .625rem .5rem',
-    position: 'relative'
+    padding: `.625rem ${ui.taskColumnPaddingInnerSmall} .5rem`,
+    position: 'relative',
+    [ui.dashBreakpoint]: {
```

Comment:
nit: it's not obvious that this is a media query.

Reply:
I think breakpoint is a pretty common semantic and speaks more to the purpose. Not all media-query declarations have to do with breakpoints. Do you have a suggestion?